config/routes.rb

```diff
@@ -47,6 +47,8 @@
   # Mission-mode-only routes
   scope ":locale/m/:mission_name", locale: /[a-z]{2}/, mission_name: /[a-z][a-z0-9]*/,
                                    defaults: {mode: "m"} do
+    mount(OData::Engine, at: "/odata/v1") if Settings.odata_api.present? || Rails.env.test?
```

Comment:
Is this a feature flag that will eventually go away?

Reply:
Yes, I'd like to be able to deploy this code without actually exposing any API-related changes yet. Let me know if you have any concerns.

Reply:
Fine here